### PR TITLE
Feature/4851 debug menu ppac

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
Added some stuff to the developer menu as requested in the tec spec:
* Display Device Time Check state
* Display API Token and Creation Date
* Generate Device Token
* Export Device Token
* Generate new API Token

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4851

## Lin to Tec Spec
https://github.com/corona-warn-app/cwa-app-tech-spec/blob/e88e3c2b6e9ed76d34ac744974e79d912b3b078f/docs/spec/data-donation-client-ppac.md#ios-developertester-menu-features

## Screenshots
Simulator Screen Shot - iPhone 12 Pro - 2021-02-02 at 22 41 20
Simulator Screen Shot - iPhone 12 Pro - 2021-02-02 at 22 41 26
Simulator Screen Shot - iPhone 12 Pro - 2021-02-02 at 22 43 29
Simulator Screen Shot - iPhone 12 Pro - 2021-02-02 at 22 43 44

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMDeviceTimeCheck/DMDeviceTimeCheckViewController.swift [hunk 1]
```diff
@@ -33,30 +33,51 @@ final class DMDeviceTimeCheckViewController: UITableViewController {
...
 	// MARK: - Private
 
 	private let viewModel: DMDeviceTimeCheckViewModel
 
 	private func setupTableView() {
+		tableView.estimatedRowHeight = 45.0
+		tableView.rowHeight = UITableView.automaticDimension
+
 		tableView.register(UINib(nibName: "DMSwitchTableViewCell", bundle: nil), forCellReuseIdentifier: "DMSwitchTableViewCell")
+		tableView.register(DMKeyValueTableViewCell.self, forCellReuseIdentifier: DMKeyValueTableViewCell.reuseIdentifier)
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Developer Menu/Features/DMDeviceTimeCheck/DMDeviceTimeCheckViewController.swift:80]: 😍

File: src/xcode/ENA/ENA/Source/Services/PPAC/__tests__/PPACServiceTest.swift [hunk 2]
```diff
@@ -116,4 +116,48 @@ class PPACServiceTest: XCTestCase {
 		wait(for: [ppacExpectation], timeout: .long)
 	}
 
+	func testGIVEN_NoStoredAPIToken_WHEN_generateAPITokenb_THEN_NewTokenCreatedAndStored() {
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Services/PPAC/__tests__/PPACServiceTest.swift:119]:
```suggestion
	func testGIVEN_NoStoredAPIToken_WHEN_generateAPIToken_THEN_NewTokenCreatedAndStored() {
```

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/ConfigureAbleCells/DMConfigureAbleCell.swift [hunk 3]
```diff
@@ -0,0 +1,23 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+#if !RELEASE
+
+import Foundation
+
+protocol DMConfigureAbleCell {
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Developer Menu/Features/ConfigureAbleCells/DMConfigureAbleCell.swift:9]:
```suggestion
protocol DMConfigureableCell {
```

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Services/PPAC/PPACService.swift [hunk 4]
```diff
@@ -44,6 +47,13 @@ class PPACService: PrivacyPreservingAccessControl {
 		deviceCheck.deviceToken(apiToken.token, completion: completion)
 	}
 
+	#if !RELEASE
+	// needed to mak it possible to get called from rhe developer menu
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Services/PPAC/PPACService.swift:51]:
```suggestion
	// needed to mak it possible to get called from the developer menu
```

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Services/PPAC/PPACService.swift (hunk 4 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Services/PPAC/PPACService.swift:51]: will fix typo it on an other PR - thanks

**Pull request merged**